src/LibraryPage.tsx

```diff
@@ -15,6 +15,7 @@ import { Book, User, Option } from "./ourtypes";
 import LibrarySearchBar from "./LibrarySearchBar";
 import { toLibrary } from "./mappers";
 import { AuthContext } from "./AuthContextProvider";
+import BookEditButton from "./libraryEditButton";
```

Comment:
We aren't 100% consistent on this, but most file names are Pascal Case too

Reply:
I've been capitalizing files whose exports are (mainly) components and keeping lowercased files that have supporting types/functions. I've also tried to name files the same as their default exports, if they have one, to make it easier to import components.